### PR TITLE
File Upload Docs

### PR DESCRIPTION
**Pull request by dblythy**
Docs For fileUpload feature

Feature: https://github.com/parse-community/parse-server/pull/7071

### CODE REVIEW

File: _includes/parse-server/file-adapters.md
Comment (mtrezza) [inline _includes/parse-server/file-adapters.md]: Instead of "starting at Parse Server V5.0.0," I think we do have a common tag that we put that indicates the applicable version.

**Review by mtrezza — CHANGES_REQUESTED** [review]

File: _includes/parse-server/file-adapters.md
Comment (mtrezza) [inline _includes/parse-server/file-adapters.md]:
```suggestion
*Available on Parse Server >=5.0.0*
```

File: _includes/parse-server/file-adapters.md
Comment (mtrezza) [inline _includes/parse-server/file-adapters.md]:
```suggestion
- Pass the encryption key via parameter `--encryptionKey` in the command line when starting Parse Server
```

File: _includes/parse-server/file-adapters.md
Comment (mtrezza) [inline _includes/parse-server/file-adapters.md]:
```suggestion
- Initialize Parse Server with `encryptionKey="PATH_TO_ENCRYPTION_KEY_FILE`.
```

File: _includes/parse-server/file-adapters.md
Comment (mtrezza) [inline _includes/parse-server/file-adapters.md]:
```suggestion
- `enableForAnonymousUser`: Enable file upload for anonymous users
- `enableForAuthenticatedUser`: Enable file upload for authenticated users
- `enableForPublic`: Enable file upload for the public, i.e. everyone
```

Comment (mtrezza) [reply _includes/parse-server/file-adapters.md]: Did we really call it `enableForPublic` instead of `enableForPublicUser`?

**Review by mtrezza — CHANGES_REQUESTED** [review]

**Comment by mtrezza** [comment]
@dblythy If you could look at my review when you have time so we can merge this

**Review by rommansabbir — APPROVED** [review]

**Review by rommansabbir — COMMENTED** [review]
Sorry, I accidentally approved the changes. @dblythy can you check the review by @mtrezza so that we can close this PR.

**Comment by dblythy** [comment]
Sure, thanks for the bump - sorry I forgot about this

File: _includes/parse-server/file-adapters.md
Comment (dblythy) [reply _includes/parse-server/file-adapters.md]: That is what we named it!

**Comment by mtrezza** [comment]
CI status doesn't appear for some reason, waiting for this issue to auto-resolve before merge.

**Review by mtrezza — APPROVED** [review]
Looks good!

**Pull request merged**
